producer.py

```diff
@@ -153,6 +152,9 @@ def package_releases_update(
         Source(**config) for config in graph.get_python_package_index_all(enabled=True)
     ]
 
+    # Class for package release messages
+    package_releases = PackageReleasesMessage()
```

Comment:
`PackageReleasedMessage`? One message should be for one package.

`package_released = PackageReleasedMessage()`

Reply:
thanks, https://github.com/thoth-station/package-releases-job/pull/449/commits/99a8fc1455c5cdbfb62b3f511551b332c36ade2c